bin/usage_update.py

```diff
@@ -11,27 +11,33 @@
 
 import settings
 from query_dao import QueryDao
+from cloudwatch_metrics import CloudwatchMetrics
 from model import AthenaQuery, QueryState, TIMESTAMP_FORMAT
 
 
 def lambda_handler(event, context):
     athena = boto3.client('athena')
     sqs = boto3.client('sqs')
     dynamodb = boto3.resource('dynamodb')
+    if settings.USE_CLOUDWATCH_ANOMALY_DETECTION:
```

Comment:
you don't use this cloudwatch instance. CloudwatchMetrics.report_query_metric uses the one defined in the CloudwatchMetrics class

Reply:
good one :D